api/general.service.ts

```diff
@@ -159,6 +161,36 @@ export class GeneralService {
             });
     }
 
+    /**
+     * Mine up to numblocks blocks immediately (before the RPC call returns).
+     * @param numBlocks How many blocks are generated immediately.
+     * @param maxtries ﻿How many iterations to try (default &#x3D; 1000000).
+     */
+    public generate(numBlocks: number, maxtries?: number, extraHttpRequestParams?: any): Observable<Array<string>> {
```

Comment:
Comment indicates maxtries defaults to 1000000 but code treats it as if it can be null.  Is this defaulting happening deeper than the generated code or did the typescript gen miss a default parameter set?

Reply:
Also could we have this as maxTries vs. "maxtries" to keep the camel-casing consistent?